<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: lightgray;
        font-family: Verdana;
        font-size: 24px;
        border-radius: 20px;
        min-width: 200px; /* Set the minimum width */
        min-height: 50px; /* Set the minimum height */
		position: absolute;
   		 left: 100px;
   		 top: 200px;
    }

    QPushButton:hover {
        background-color: rgb(203, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: lightgray;
        font-family: Verdana;
        font-size: 24px;
        border-radius: 20px;
        min-width: 200px; /* Set the minimum width */
        min-height: 50px; /* Set the minimum height */
		position: absolute;
   		 left: 100px;
   		 top: 200px;
    }

    QPushButton:hover {
        background-color: rgb(203, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>700</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: lightgray;
        font-family: Verdana;
        font-size: 24px;
        border-radius: 20px;
        min-width: 200px; /* Set the minimum width */
        min-height: 50px; /* Set the minimum height */
		position: absolute;
   		 left: 100px;
   		 top: 200px;
    }

    QPushButton:hover {
        background-color: rgb(203, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExtrasButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: lightgray;
        font-family: Verdana;
        font-size: 24px;
        border-radius: 20px;
        min-width: 200px; /* Set the minimum width */
        min-height: 50px; /* Set the minimum height */
		position: absolute;
   		 left: 100px;
   		 top: 200px;
    }

    QPushButton:hover {
        background-color: rgb(203, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Extras</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
